### PR TITLE
Update constants requirement

### PR DESCRIPTION
**Pull request by designatednerd**
This commit updates the constants requirement to the new version of ILGHttpConstants, which will allow for better Swift Compatibility. Gotta update this in order to update a few things that depend on it. 

@vokal-isaac let me know if you'd prefer I bump this to 3.0.0 - since it doesn't change anything within this repo but it does change a big dependency, I figured a minor patch was probably the best course of action.

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal-isaac @bryanluby @brockboland et al, review in the morning plz? 😄

**Comment by designatednerd** [comment]
(sorry for the big diff, the last pod install was run on 0.39, so it was gonna get nuked from orbit anyway 😛 )

**Comment by brockboland** [comment]
If @bryanluby and @vokal-isaac have no objections, I'll merge this by EOD.

File: VOKMockUrlProtocol.podspec [hunk 1]
```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.source_files = 'VOKMockUrlProtocol.[hm]'
-  s.dependency 'ILGHttpConstants', '~> 1.0.0'
+  s.dependency 'ILGHttpConstants', '~> 2.0.0'
```
Comment (vokal-isaac) [inline VOKMockUrlProtocol.podspec]: Maybe this should be looser, like `~> 2.0`?

Comment (designatednerd) [inline VOKMockUrlProtocol.podspec]: I figured I'd leave it at the same level as before - I feel like there's not going to be a huge need for people to have wildly divergent values for this.

Comment (designatednerd) [inline VOKMockUrlProtocol.podspec]: Not this PR, since it's a major version upgrade.

**Comment by brockboland** [comment]
SLGTM

**Comment by designatednerd** [comment]
🎉 thank you!

**Comment by brockboland** [comment]
@designatednerd cocoapods has been updated https://cocoapods.org/pods/VOKMockUrlProtocol

**Comment by designatednerd** [comment]
sweeeeet thanks again @brockboland

**Pull request merged**